packages/pwa/app/routes.jsx

```diff
@@ -37,41 +37,36 @@ const PageNotFound = loadable(() => import('./pages/page-not-found'))
 
 const routes = [
     {
-        path: '/',
+        path: '/:siteIdAlias?',
```

Comment:
I'm starting to think that it's going to be easier for us to always ave the site and locale show in the path, with the exception of the home path ("/"). The reason I say this is that when we try to do things like, get the locale from the path, we'll now have to take into account if we are showing the siteid in our url or not and that can get kind of clunky. 

If you think about it, the hostname is independent of the site name. (I'm talking about option 2 here)... potentially someone could run 2 sites on hostname A and 2 sites on hostname B, so it's important that the site name / alias is always in the path. For this hypothetical situation.

Reply:
yea, I think that is a good idea. 

However, what if partners dev don't want to have the site alias on their URL? This would potentially apply to the locale shortcode as well.